src/ImageSharp.Drawing/Text/TextGraphicsOptions.cs

```diff
@@ -39,6 +44,7 @@ public TextGraphicsOptions(bool enableAntialiasing)
             this.Antialias = enableAntialiasing;
             this.ApplyKerning = true;
             this.TabWidth = 4;
+            this.AntialiasSubpixelDepth = 16;
```

Comment:
Entirely subjectively it was the point I felt that that the image quality at the size feature I was testing at didn't seem to degrade any.

You'll all so notice the Text has a different default to general shapes, the reason for this is the glyphs will most likely be rendered at smaller sizes that general shapes and they will have more finer detail what needs preserving.